library/swift/src/BUILD

```diff
@@ -10,10 +10,14 @@ swift_static_framework(
     name = "ios_framework",
     srcs = [
         "Envoy.swift",
+        "EnvoyResult.swift",
```

Comment:
Hah, should we keep `Envoy` as a prefix here as well? Feels a bit redundant since we have the module/package naming

WDYT?

Reply:
Ah yea I meant to rename this to just `Result`